vendors/pc/boards/windows/aws_demos/config_files/aws_ggd_config.h

```diff
@@ -32,15 +32,39 @@
 #ifndef _AWS_GGD_CONFIG_H_
 #define _AWS_GGD_CONFIG_H_
 
+/**************************************************/
```

Comment:
This should probably be changed for all the boards and not just Windows, right?

Reply:
Yeah